src/main/java/wolox/training/models/Book.java

```diff
@@ -66,6 +67,7 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
+        Preconditions.checkNotNull(author);
```

Comment:
You should also validate that the string is not empty (the same goes for all other strings in the project) :) 

You may also consider adding a descriptive error message, such as "The author must not be null or empty". I recommend creating a class that contains all message and error constants as static fields. Let me know if you need help with this 😄